README.md

```diff
@@ -26,69 +26,61 @@ What's missing?
 ---------------
 
 Lots of things!
-- An ament build type for Cargo. The current examples use CMake to install and build the binaries... and it's really ugly.
 - Component nodes
 - Clients and services
 - Tests
 - Documentation
 
 ### Limitations
 
-- messages are deep-copied and this can be terribly inefficient for big messages like images; the current solution leverages C typesupport implementations and might benefits from a direct serialization/deserialization
-- the current solution for crates export with CMake is not very robust
+- messages are deep-copied and this can be terribly inefficient for big messages like images; the current solution leverages C typesupport implementations and might benefit from a direct serialization/deserialization
 - `rclrs` interface is very limited for now and might not be so much idiomatic yet, any help and suggestion on the interface would be greatly appreciated
 - due to the current ROS2 support of non-default clients, packages containing definitions of messages used in Rust crates must be present in the current workspace; otherwise message crates generation won't be triggered
 
 Sounds great, how can I try this out?
 -------------------------------------
 
-You can build and run the example using the included Dockerfile: 
+The following steps were last tested on Ubuntu 20.04. Here, the Foxy distribution of ROS 2 is used, but newer distributions can be used by simply replacing 'foxy' with the distribution name.
 
 ```
-git clone https://github.com/ros2-rust/ros2_rust.git
-docker build --tag ros2:rust .
-docker run -it --rm ros2:rust
-ros2 run rclrs_examples rclrs_publisher &
-ros2 run rclrs_examples rclrs_subscriber
-
-```
-
-Or do so manually as summarized in the steps below:
-
-> The following steps were last tested on Ubuntu 20.04.
-
-```
-# first, install vcstool from PyPI or apt:
+# First, make sure to have ROS 2 and vcstool installed:
 # sudo apt install ros-foxy-desktop ros-foxy-test-interface-files python3-vcstool libclang-dev clang
+# vcstool can alternatively be installed from PyPI:
 # pip install vcstool
 
-mkdir -p ~/ros2_rust_ws/src/ros2-rust
-cd ~/ros2_rust_ws/src/ros2-rust
-git clone https://github.com/ros2-rust/ros2_rust.git
-cd ../../
-vcs import src < src/ros2-rust/ros2_rust/ros2_rust_foxy.repos
+# In your workspace directory (ideally an empty one), run 
+mkdir src
+git clone https://github.com/ros2-rust/ros2_rust.git src/ros2_rust
+vcs import src < src/ros2_rust/ros2_rust_foxy.repos
 source /opt/ros/foxy/setup.sh
-colcon build
+colcon build --install-base install_colcon --packages-up-to colcon-ros-cargo
```

Comment:
A separate workspace for `colcon` is no longer needed because we can install `colcon-ros-cargo` via `pip`

Reply:
Done.